File: Question3.c

//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//  File Name       : Question_3.c
//  Description     : Accept one number from user and print its number line
//  Author          : lina vijay patil
//  Date            : 22/10/2025
//
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
#include<stdio.h>
void Display(iNo)
{ 
    int iCnt = 0;
    
    if(iNo < 0)
    {
        iNo = -iNo;
    }
    for (iCnt = -iNo ; iCnt <= iNo ; iCnt ++)
    {
        printf("%d\t",iCnt);
    }
}
int main()
{
   int iValue = 0 ;
   
   printf("Enter Your number\n");
   scanf("%d",&iValue);

   Display(iValue);

  return 0;


}

File: Question3.java

////////////////////////////////////////////////////////////
// 
//  File name :     Question3.java
//  Description :   To find factorial of a number using for loop
//  Author :        lina vijay patil
//  Date :          03/11/2025
//
////////////////////////////////////////////////////////////

class Logic
{
    int CalculateFactorial(int iNo)
    {
        int iFact = 1;

        for (int iCnt = 1; iCnt <= iNo; iCnt++)
        {
            iFact = iFact * iCnt;
        }
        return iFact;
    }
}

class Quetion3
{
    public static void main(String A[])
    {
        Logic lobj = new Logic();
    
        int iRet = lobj.CalculateFactorial(5);

        System.out.println("Factorial of is: " + iRet);
    }
}
